AudioLink/Shaders/AudioLink.shader

```diff
@@ -525,6 +548,17 @@ Shader "AudioLink/Internal/AudioLink"
                             return AudioLinkGetSelfPixelData(ALPASS_CCCOLORS+uint2(1+coordinateLocal.x,0));
                         }
                     }
+                    else if( coordinateLocal.x == 4 )
```

Comment:
I'm assuming you made sure this doesn't overlap on the CRT with any of the other newer features? I don't really have a good overview myself

Reply:
I will need to triple check - but that's what the dev branch is for!